test/thermodynamics/irradiation_of_black_body_from_temperature_test.py

```diff
@@ -0,0 +1,30 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    convert_to,
+    Quantity,
+    SI,
+)
+from symplyphysics.laws.thermodynamics import irradiation_of_black_body_from_temperature as stefan_boltzmann_law
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    t = Quantity(20 * units.kelvin)
+    Args = namedtuple("Args", ["t"])
+    return Args(t=t)
+
+def test_basic_radiant_heat_energy(test_args):
+    result = stefan_boltzmann_law.calculate_irradiance(test_args.t)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.power/units.length**2)
+    result_radiant_heat_energy_joule = convert_to(result, units.joule/units.meter**2/units.second).evalf(6)
```

Comment:
It's not energy and not joule.
units.joule / units.second is also known as watt.